apps/labs/posts/labs-annual-report-2024.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Quansight Labs Annual Report 2024: Year of focus and execution'
+authors: [pavithra-eswaramoorthy, tania-allard, irina-fumarel, ralf-gommers]
+published: March 19, 2025
+description: 'Presenting our 2024 annual report! Read about our open source project and community highlights, initiatives, and work culture.'
+category: [Community, PyData ecosystem]
+featuredImage:
+  src: /posts/labs-annual-report-2024/featured.png
+  alt: 'A line-style illustration that depicts an urban street scene with people with disabilities. On the left, a person in a wheelchair is being assisted to cross the street. In the center, two people walk arm-in-arm in the crosswalk. On the right, a person with a cane and a guide dog walks along the sidewalk. The background shows a simplified city skyline with buildings, trees, a traffic light, and a bus stop sign. The image aims to depict accessibility and inclusion in urban environments.'
+hero:
+  imageSrc: /posts/labs-annual-report-2024/hero.png
+imageAlt: 'A line-style illustration that depicts an urban street scene with people with disabilities. On the left, a person in a wheelchair is being assisted to cross the street. In the center, two people walk arm-in-arm in the crosswalk. On the right, a person with a cane and a guide dog walks along the sidewalk. The background shows a simplified city skyline with buildings, trees, a traffic light, and a bus stop sign. The image aims to depict accessibility and inclusion in urban environments.'
+---
+
+We’re pleased to share the Quansight Labs Annual Report 2024!
+
+As we settle into the new year, we’re taking a moment to reflect on 2024—a year filled with impactful projects, meaningful collaborations, and continued growth in our mission to sustain and advance community-driven open-source projects.
+
+Our annual report offers an in-depth look at our work over the past year, covering key initiatives, projects and community updates, financial insights, and team growth.
+
+<p align="center">
+  <a href="https://labs.quansight.org/annual-reports/quansight-labs-annual-report-2024">
+    <b>Click to read and download the full report</b>
+  </a>
+</p>
+
+The following graphic presents notable highlights from the report.
+
```

Comment:
Since we have the screen reader-only markup now, I wonder if we should remove this sentence
```suggestion
```

Reply:
I think we can keep it. The transition to the graphic feels too abrupt otherwise IMO